types/react/index.d.ts

```diff
@@ -38,6 +38,7 @@
 import * as CSS from 'csstype';
 import * as PropTypes from 'prop-types';
 import { Interaction as SchedulerInteraction } from 'scheduler/tracing';
+import 'trusted-types';
```

Comment:
I think we should rather use the same approach we do with the dom related stuff: use an empty interface for `TrustedHTML` and let users take care of the proper type. We shouldn't use the polyfill directly here.

For more guidance check how we test the other DOM interfaces.

Reply:
Ah, thanks, I thought `trusted-types` were the global types to be used. https://github.com/microsoft/TypeScript-DOM-lib-generator/pull/1246 is stalled, so they're not available globally yet. I'll try to take a look.